src/FSharp.AWS.DynamoDB/Expression/ConditionalExpr.fs

```diff
@@ -151,10 +147,7 @@ let extractKeyCondition (qExpr: QueryExpr) =
                     | (ks, KeyType.Hash) -> Some ks
                     | _ -> None)
             | Some rk ->
-                Seq.joinBy
-                    (fun (ks, kt) (ks', kt') -> kt = KeyType.Hash && kt' = KeyType.Range && ks = ks')
-                    hk.KeySchemata
-                    rk.KeySchemata
+                Seq.joinBy (fun (ks, kt) (ks', kt') -> kt = KeyType.Hash && kt' = KeyType.Range && ks = ks') hk.KeySchemata rk.KeySchemata
```

Comment:
left was better but fair enough
(should actuallt change to do (hk..., rk...) ||> Seq.joinBy